src/util/file_watch.c

```diff
@@ -122,6 +122,9 @@ static int watched_file_inotify_cb(const char *filename,
                                   void *pvt)
 {
     struct file_watch_ctx *fw_ctx;
+    char received[PATH_MAX + 1];
+    char expected[PATH_MAX + 1];
```

Comment:
Maybe too old school but I'd made those static to avoid "big" chunks on stack.

Reply:
Good idea.